.github/workflows/precompiled.yaml

```diff
@@ -20,19 +20,35 @@ on:
     - cron: '00 09 * * *' # scheduled job
 
 jobs:
+  set-driver-version-matrix:
```

Comment:
What is the motivation for dynamically determining the matrix of driver branches? In my opinion, it is clearer if we just maintained a a hardcoded list of kernel flavors and driver branches.

Reply:
I think it is okay to have this job defined, but I think it would be easier to read if we removed the shell script which determines the driver branches. Let's define both `driver_versions` and `kernel_flavors` statically here.